x/oracle/client/cli/txUpdateOracleInfo.go

```diff
@@ -0,0 +1,52 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdUpdateOracleInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-oracle-info [endpoint] [oracle-commission-rate]",
```

Comment:
I think this is similar. How about change to using flag?

Reply:
How about using endpoint by argument and oracle commission rate to using flag?

Reply:
Thanks, I changed it using flag ([ab40af0](https://github.com/medibloc/panacea-core/pull/540/commits/ab40af0819b45dee477db3cb1317e287c0a0efdd)).